docs/howto/manage-charms.rst

```diff
@@ -234,12 +234,17 @@ charm's project file, specify an item under the :ref:`links.website
 Add docs
 ~~~~~~~~
 
-If you publish your charm on Charmhub, reference documentation about the charm's
-resources, actions, configurations, relations, and libraries is extracted automatically.
-However, you should also aim to add further docs, e.g., a tutorial, how-to guides, etc.
-To provide a link to these docs, in your charm's project file, specify a
-value for the :ref:`links.documentation <charmcraft-yaml-key-documentation>` key. Note
-that at present this must be a Discourse page. E.g.,
+When you publish your charm on Charmhub, reference documentation about the charm's
+resources, actions, configurations, relations, and libraries is generated and 
+published automatically in respective tabs.
+Use the Description tab to add additional documentation, such as tutorials or 
```

Comment:
```suggestion

On the **Description** tab, add additional documentation, such as tutorials or 
```

I think this would read better with a paragraph break here.

Reply:
Fixed in https://github.com/canonical/charmcraft/pull/2253/commits/8ef59e0da29fd0535a9d45fe03fa8ec90af3ef13